app/errors.py

```diff
@@ -113,3 +114,45 @@ def internal_server_error(e):
         e = getattr(e, "original_exception", e)
         current_app.logger.exception(e)
         return jsonify(result="error", message="Internal server error"), 500
+
+
+class TooManyRequestsError(InvalidRequest):
```

Comment:
This class does not appear to be used

Reply:
Thanks for flagging, @jskinne3!  While that might be the case, I could see it being used in the not-too-distant future when we revisit message limits.  We can leave in for now and remove it later if it turns out we truly don't need it.